sdk/keyvault/keyvault/README.md

```diff
@@ -102,4 +102,4 @@ See https://github.com/Azure/ms-rest-browserauth to learn how to authenticate to
 - [Microsoft Azure SDK for Javascript](https://github.com/Azure/azure-sdk-for-js)
 
 
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js%2Fpackages%2F%40azure%2Fkeyvault%2FREADME.png)
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js/sdk/keyvault/keyvault/README.png)
```

Comment:
@scbedd do these paths need to be escaped?

Reply:
100% they do.

Reply:
I just looked at the raw for the others that were moved.

https://raw.githubusercontent.com/Azure/azure-sdk-for-js/master/sdk/template/template/README.md
https://raw.githubusercontent.com/Azure/azure-sdk-for-js/master/sdk/servicebus/service-bus/README.md
https://raw.githubusercontent.com/Azure/azure-sdk-for-js/master/sdk/servicebus/service-bus/README.md

Also need to be updated back to the encoding method.

Reply:
Thanks!